lib/index.d.ts

```diff
@@ -481,25 +481,25 @@ export namespace api {
         public clear?: (api.Clear|null);
 
         /** Command eval. */
-        public eval: string;
+        public eval?: (string|null);
```

Comment:
hmm these changes shouldn't be here: concrete message types always have all primitive values defined and populated with an "empty value".

most likely the scripts to fix these are no longer catching this due to the addition of the `--root` parameter, so they may need a little tweaking to generate the correct code. (i just wish that upstream would accept some of the PRs i've sent to avoid the local scripts u_u)